compiler/rustc_mir/src/interpret/validity.rs

```diff
@@ -104,7 +104,7 @@ pub enum PathElem {
     Field(Symbol),
     Variant(Symbol),
     GeneratorState(VariantIdx),
-    CapturedVar(Symbol),
+    CapturedPlace(Symbol),
```

Comment:
This should probably store a vector of Symbols, or something like that. Because we want to describe the CapturedPlace Now